### PR TITLE
Stand-alone transaction in dev guides.

### LINKED ISSUE #729: Explain how the AEA uses the blockchains
**Is your feature request related to a problem? Please describe.**
Currently, the demos include examples how the AEAs use the blockchain. But there is no standalone explanation.

**Describe the solution you'd like**
Explain how AEAs interact with the blockchain. Also add a section in the build a skill guide on using the ledger to do a simple transaction.

### PR DESCRIPTION
**Pull request by Totoual**
## Proposed changes

Adds an example of using `LedgerApis` and `Wallets` to create some wealth and sends a transaction in the Fetch.ai network

## Fixes

#729 
#738 2/2

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/standalone-transaction.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+In this guide, we will generate some wealth for the `Fetch.ai testnet` and create a standalone transaction. After the completion of the transaction,
+we get the transaction digest. With this we can search the transaction it in the <a href='https://explore-testnet.fetch.ai'>block explorer</a>
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: search for the transaction on the

File: docs/standalone-transaction.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Create the wallets
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: First create the private key files.

File: docs/standalone-transaction.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+We will need to create two different addresses for this demo. To do this we instantiate two different wallets 
+
+```python
+    wallet_1 = Wallet({FETCHAI: FETCHAI_PRIVATE_KEY_FILE})
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: use dynamically created key files, also add the import statements at the beginning for all required modules and constants

File: docs/standalone-transaction.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+Finally, we create a transaction that sends the funds to the `wallet_2`
+
+```python
+  ledger_apis.apis['fetchai'].send_transaction(crypto=wallet_1.crypto_objects.get(FETCHAI),
+                                                 destination_address=wallet_2.addresses.get(FETCHAI),
+                                                 amount=1,
+                                                 tx_fee=1,
+                                                 tx_nonce="this_is_a_transaction_nonce",
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: use the helper method to generate the nonce

File: docs/standalone-transaction.md (hunk 3 above)
Comment (Totoual) [reply docs/standalone-transaction.md]: I am not sure what you mean here.

Comment (Totoual) [reply docs/standalone-transaction.md]: I think I found it.

File: docs/standalone-transaction.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
+In this guide, we will generate some wealth for the `Fetch.ai testnet` and create a standalone transaction. After the completion of the transaction,
+we get the transaction digest. With this we can search the transaction on the <a href='https://explore-testnet.fetch.ai'>block explorer</a>
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: we can search for the

File: docs/standalone-transaction.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+## Create the private keys
+
+Firstly, we will create the private key files.
+
+```python
+from aea.crypto.helpers import _create_fetchai_private_key
+
+FETCHAI_PRIVATE_KEY_FILE_1 = "fet_private_key_1.txt"
+FETCHAI_PRIVATE_KEY_FILE_2 = "fet_private_key_2.txt"
+# Create a private key
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: Create private keys

File: docs/standalone-transaction.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+```python
+
+from aea.crypto.fetchai import FETCHAI
+from aea.crypto.wallet import Wallet
+
+FETCHAI_PRIVATE_KEY_FILE_1 = "fet_private_key_1.txt"
+FETCHAI_PRIVATE_KEY_FILE_2 = "fet_private_key_2.txt"
+
+# Set up the wallet
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: Set up the wallets

File: docs/standalone-transaction.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+# Generate some wealth
+_try_generate_testnet_wealth(FETCHAI, wallet_1.addresses[FETCHAI])
+```
+
+Finally, we create a transaction that sends the funds to the `wallet_2`
+
+```python
+from aea.crypto.ledger_apis import LedgerApis
+
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: you're missing instantiation of the ledger apis

File: docs/standalone-transaction.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+Since we want to send funds from `wallet_1` to `wallet_2`, we need to generate some wealth for the `wallet_1`. We can
+do this with the following code
+```python
+from aea.crypto.helpers import _try_generate_testnet_wealth
+
+# Generate some wealth
+_try_generate_testnet_wealth(FETCHAI, wallet_1.addresses[FETCHAI])
+```
+
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: New section: ## Send transaction

File: docs/standalone-transaction.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+Finally, we create a transaction that sends the funds to the `wallet_2`
+
+```python
+from aea.crypto.ledger_apis import LedgerApis
+
+# Create the transaction and send it to the ledger.
+ledger_apis.apis['fetchai'].send_transaction(crypto=wallet_1.crypto_objects.get(FETCHAI),
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: two lines:
ledger_api = ledger_apis.apis['fetchai']
ledger_api.send_transaction(..

File: docs/standalone-transaction.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+                                             destination_address=wallet_2.addresses.get(FETCHAI),
+                                             amount=1,
+                                             tx_fee=1,
+                                             tx_nonce=ledger_apis.apis.get(FETCHAI).generate_tx_nonce(wallet_2.addresses.get(FETCHAI),
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: have a line above
tx_nonce = ...
then use the variable here.

File: docs/standalone-transaction.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+                                                                                                      wallet_1.addresses.get(FETCHAI)),
+                                             )
+```
+
```
Comment (DavidMinarsch) [inline docs/standalone-transaction.md]: Where is the digest to confirm the tx?

**Comment by DavidMinarsch** [comment]
Looks good, what about Part 2?

**Comment by Totoual** [comment]
Part2 is under development.

**Pull request merged**